jvm/core/src/main/scala/org/scalatest/tools/Runner.scala

```diff
@@ -1489,8 +1489,17 @@ object Runner {
         // getDispatchReporter may complete abruptly with an exception, if there is an problem trying to load
         // or instantiate a custom reporter class.
         case ex: Throwable => {
-          System.err.println(Resources.bigProblemsMaybeCustomReporter)
```

Comment:
I think this is an improvement, for sure, but this is another catch of Throwable. Should we be using NonFatal here also?

Reply:
@bvenners The Throwable was there before this PR, I think we did want to catch everything, as here we're going to fire RunAborted event to stop the run properly, if it is a fatal exception (like out of memory) that can cause the following code to not able to execute, I think it will be blowing up again here.